olaaf_django/sync_hashes.py

```diff
@@ -124,21 +124,31 @@ def _sync_hashes_for_publication(repo, publication, publication_commits, chrome_
 
 
 def _find_all_publication_branches(repo):
+  local_branches = [branch.name for branch in repo.branches]
   remote_branches = repo.git.branch('-r')
-  if remote_branches:
-    remote_branches = [branch.strip() for branch in remote_branches.split('\n') if 'HEAD' not in branch]
-  local_branches = repo.branches
+  remote_branches = [
+    branch.strip().split('/', 1)[1] for branch in remote_branches.split('\n')
+    if 'HEAD' not in branch
+  ] if remote_branches else []
+
+  # get publication branches
+  branches = [
+    b for b in set(remote_branches + local_branches)
+    if _check_if_valid_publication_branch_name(b)
+  ]
+  # sort by 'publication/2019-01-01' first and then '-01' index
+  branches = sorted(branches, key=lambda x: (x[:22], x[22:]))
```

Comment:
We could use a regex here. We already have the branch name regex defined. Not a crucial thing though, this should work as well

Reply:
Yup, this was just quicker for me to implement :)